hardware.inc

```diff
@@ -761,19 +777,136 @@ def B_RAMB_RUMBLE equ 3 ; (MBC5 and MBC7 only) enable the rumble motor (if any)
         def RAMB_RUMBLE_OFF equ 0 << B_RAMB_RUMBLE
         def RAMB_RUMBLE_ON  equ 1 << B_RAMB_RUMBLE
 
+
+; ** MBC1 and MMM01 only ******************************************************
+
+; -- BMODE ($6000-$7FFF) ------------------------------------------------------
+; Banking mode select [wo]
+def rBMODE equ $6000
+
+def BMODE_SIMPLE   equ $00 ; locks ROMB and RAMB to bank 0
+def BMODE_ADVANCED equ $01 ; allows bank-switching with RAMB
+
+
+; ** MBC2 only ****************************************************************
+
+; -- ROM2B ($0000-$3FFF) ------------------------------------------------------
+; ROM bank number [wo]
+def rROM2B equ $2100
```

Comment:
If the range for `rROM2B` is $0000-$3FFF, how come the canonical address is $2100? Just curious.

Reply:
nvm, right after I posted I saw that you already talked about this:

> The only register name that can ever be, not just irrelevant, but _wrong_, is "`ROMB`" for the ROM bank number register. It's commonly $2000, but for MBC2 should be $2100 (or any value with bit 8 set, $0100, $0300, etc).

Does that mean the `; -- ROM2B ($0000-$3FFF)` comment is a little wrong?

Reply:
Yeah, the comment could be more precise. @ISSOtm Any preference for bikeshedding this?

- `; -- ROM2B ($0000-$3FFF with bit 8 set) ---------------------------------------`
- `; -- ROM2B ($0100-$01FF, $0300-$03FF, $0500-$05FF, ..., $3F00-$3FFF) ----------`
- `; -- ROM2B (%00xxxxx1xxxxxxxx) ------------------------------------------------`

Reply:
I'd vote for your first option.

Reply:
Same.

Reply:
Okay!